experiments/download_landsat.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import numpy as np
+import os
+import ee
+
+from geeS2downloader import GEES2Downloader
+
+def download_data(args):
+    
+    # initialize seed
+    os.makedirs(args.save_path, exist_ok=True)
+
+    # initialize ee
+    ee.Initialize()
+
+    downloader = GEES2Downloader()
+
+    conus = ee.FeatureCollection("TIGER/2018/States")
```

Comment:
Using this entire Feature Collection is making it suuuuper slow. I tried replacing it with a rough rectangle polygon over the US from USGS but that is too rough because you also get some Canada. Will have to look for alternatives to filter the bounds.